tutorials/python/python-type-driven-development/tutorial.md

```diff
@@ -0,0 +1,218 @@
+---
+title: How to start using types in Python with Mypy
+published: 2020-01-29
+tags: Python, tutorial, typing, types, tdd
+language: Python
```

Comment:
Does this work the same if you change it to `python`? There's another tutorial about Python that uses the lowercase version, but I don't remember if that does something different.

Reply:
Changed to `python` to keep consistency.